pulp_rpm/app/modulemd.py

```diff
@@ -26,7 +27,7 @@ def modules_packages(modules):
     current_modules = Modulemd.objects \
         .filter(pk__in=version.content.filter(pulp_type="rpm.modulemd"))
 
-    if previous_version:
+    if previous_version.number:
```

Comment:
I'm confused as to what this is doing now.  Is this a special case for the first repository version? Do we really need it?  Can it ever be ```None```?

We should document the ```previous_version``` argument as well

Reply:
I think the main goal was to handle a special case of the first version.

It doesn't seem that the previous version can be None: https://github.com/pulp/pulp_rpm/blob/15f043b0139db7cd3aa49ce5af2441ca8e44c039/pulp_rpm/app/models/repository.py#L104-L111
Looking at the code ^, it's another refactor - we need to start using what core provides, it was from the times when core didn't provide the last version number.